### ISSUE TITLE
DB: user

### ISSUE THREAD

**Issue by HexMakina** [SimplyV/MyTechOnWeb#3]
- un champ `passworduser` ?
- les champs suivant (adresse) sont préfixés par `user`

donc manquement en cohérence de nommage et préfixe/suffixe inutile, non ?

**Comment by SimplyV** [comment]
> 

Effectivement, ça n'avait pas beaucoup de sens sachant qu'on est déjà dans la table user. C'est réglé !
